src/main/java/com/github/packageurl/PackageURL.java

```diff
@@ -248,7 +244,7 @@ public PackageURL(
      * @param subpath the subpath string
      * @throws MalformedPackageURLException if parsing fails
      * @throws NullPointerException if {@code type} or {@code name} are {@code null}
-     * @since 1.6.0
+     * @since 2.0.0
      */
     public PackageURL(
```

Comment:
With the introduction of this constructor, the deprecated constructor that takes `TreeMap` above should also be removed.

Reply:
I'll remove this in another PR.

Reply:
This was added to #219.